src/tests/EdFi.DataManagementService.Tests.E2E/Management/ContainerSetup.cs

```diff
@@ -45,23 +47,19 @@ public static async Task<string> SetupDataManagement()
             .WithImage(apiImageName)
             .WithPortBinding(8080)
             .WithEnvironment("NEED_DATABASE_SETUP", "true")
-            .WithEnvironment(
-                "DATABASE_CONNECTION_STRING",
-                "host=dmsdb;port=5432;username=postgres;password=P@ssw0rd;database=edfi_datamanagementservice;"
-            )
-            .WithEnvironment("POSTGRES_ADMIN_USER", pgAdminUser)
-            .WithEnvironment("POSTGRES_ADMIN_PASSWORD", pgAdminPassword)
-            .WithEnvironment("POSTGRES_PORT", "5432")
-            .WithEnvironment("POSTGRES_HOST", dbContainerName)
+            .WithEnvironment("DATABASE_CONNECTION_STRING", connectionString)
+            .WithEnvironment("DATABASE_CONNECTION_STRING_ADMIN", connectionString)
             .WithEnvironment("LOG_LEVEL", "Debug")
-            .WithEnvironment("OAUTH_TOKEN_ENDPOINT", "http://127.0.0.1:8080/oauth/token")
+            .WithEnvironment("OAUTH_TOKEN_ENDPOINT", $"http://127.0.0.1:8080/oauth/token")
```

Comment:
Oops, that `$` ended up being unnecessary, though if there are no other commits necessary then I'm not going to fix this and trigger re-execution of tests.

I was trying to change the port from 8080 to something else. I never got it to succeed. There is an 8080 somewhere else that I can't find.